src/stores/widgets/PvGeneratorWidgetStore.ts

```diff
@@ -3,9 +3,18 @@ import {IOptionProps} from "@blueprintjs/core";
 import {CARTA} from "carta-protobuf";
 import {AppStore} from "stores";
 import {RegionWidgetStore, RegionsType, RegionId} from "./RegionWidgetStore";
+import {SpectralSystem} from "models";
+
+export enum PVAxis {
+    SPATIAL = "Spatial",
+    SPECTRAL = "Spectral"
+}
 
 export class PvGeneratorWidgetStore extends RegionWidgetStore {
     @observable width: number;
+    @observable reverse: boolean;
+    @observable keep: boolean;
+    @observable range: CARTA.IIntBounds = {min: null, max: null};
```

Comment:
<img width="970" alt="image" src="https://user-images.githubusercontent.com/43841102/190358483-16a35f84-50fa-46fe-abe6-435933efd280.png">
I think it would be useful if we set the entire spectral range as default, like in the moment generator.

Reply:
I realized that the moment generator range input isn't updating when we change the coordinate. I fixed it I will reverse it if it is not a desirable behaviour.

Reply:
The input fields now should display the entire spectral range in selected coordinate as default.